src/components/DocumentationTopic.vue

```diff
@@ -11,9 +11,9 @@
 <template>
   <div class="doc-topic">
     <main class="main" id="main" role="main" tabindex="0">
-      <DocumentationHero :type="symbolKind || role">
+      <DocumentationHero :type="symbolKind || role" :enableHero="enableHero">
         <slot name="above-title" />
-        <Title :eyebrow="roleHeading">{{ title }}</Title>
+        <Title :eyebrow="roleHeading" :style=heroTitleStyle>{{ title }}</Title>
```

Comment:
I don't think manually updating the `style` attribute is necessary here and probably not preferred. I'm actually a little surprised that the SASS functions you're using would actually work in that manner, but maybe they do.

As an alternative, I would suggest maybe just applying a class to the hero element based on the boolean flag you already have. Then, you can just update the CSS for components like the hero as normal, and they would have rules like so:

```css
.documentation-hero {
  &.enhanced-background {
    --title-color: light-color(fill);
    --border-width: 0;
    /* etc */
  }
}
```

Reply:
I got a little further in the PR, and it seems like you already have a class like this that you can already use to do this :)

Reply:
I used this method for styling the Title component, but not sure if I can or should do the same for the border width. See: https://github.com/dobromir-hristov/swift-docc-render/pull/10#discussion_r824553880